pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
> And for the package name, I think the convention in this project is using under score.

Where is the convention documented. Is there any other pkgs has underscore in names in k8s?